build/ci/evergreen.yml

```diff
@@ -175,6 +175,39 @@ functions:
         args: ['-h', 'hosts.json']
         env:
           XDG_CONFIG_HOME: ${workdir}
+  "set-up cloud manager":
+    - command: subprocess.exec
+      type: setup
+      params:
+        add_to_path:
+          - ${go_bin}
+          - ../../bin
+        working_dir: src/github.com/mongodb/mongocli/build/ci
+        include_expansions_in_env:
+          - cloud_manager_service
+          - revision
+        binary: ./set-up-cloud-manager.sh
+        args: ['-h', 'hosts.json']
+        env:
+          XDG_CONFIG_HOME: ${workdir}
+          MCLI_ORG_ID: ${cloud_manager_org_id}
+          MCLI_PROJECT_ID: ${cloud_manager_project_id}
+          MCLI_PRIVATE_API_KEY: ${cloud_manager_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${cloud_manager_public_api_key}
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_SERVICE: cloud-manager
+  "clean-up cloud manager":
+    - command: subprocess.exec
+      type: setup
+      params:
+        add_to_path:
+          - ${go_bin}
+          - ../../bin
```

Comment:
[Question for my own learning] Why are we using relative path like this `../../bin` and not relative with respect to `${workdir}/`?

Reply:
the `../../` is relative to `workdir` that is configured to be `src/github.com/mongodb/mongocli/build/ci`, in other words, resulting destination will be `src/github.com/mongodb/mongocli/bin`, I've tried the path straight away and for some reason it did not work.